eng/testing/RunnerTemplate.sh

```diff
@@ -110,6 +136,10 @@ function print_info_from_core_file {
   echo "Printing info from core file $core_file_name"
   print_info_from_core_file_using_gdb $core_file_name $executable_name
   print_info_from_core_file_using_lldb $core_file_name $executable_name
+
+  if [ $lldb_succeeded -eq 0 ] && [ $gdb_succeeded -eq 0 ]; then
+    print_info_from_core_file_using_XUnitLogChecker $core_file_name
+  fi
 }
```

Comment:
Below in `BEGIN Core File Setup` we want to set the `DOTNET` variables that @hoyosjs was recommending.